### PR TITLE
added two possible solutions to dplyr challenge 2

### PR DESCRIPTION
**Pull request by liz-is**
Added solutions for the second part of the challenge, as suggested in issue #230.

I've added solutions with and without the use of `arrange()`, as learners haven't seen this function during the lesson.

### CODE REVIEW

**Comment by fmichonneau** [comment]
Personally, I think it would be best to stick to a dplyr approach for a solution to this challenge.
Another possibility would be to do: `... %>% filter(mean_lifeExp == min(mean_lifeExp) | mean_lifeExp == max(mean_lifeExp))`

File: _episodes_rmd/13-dplyr.Rmd [hunk 1]
```diff
@@ -181,6 +181,17 @@ even better.
 > >lifeExp_bycountry <- gapminder %>%
 > >    group_by(country) %>%
 > >    summarize(mean_lifeExp=mean(lifeExp))
+> >
+> >lifeExp_bycountry$country[which.max(lifeExp_bycountry$mean_lifeExp)]
+> >lifeExp_bycountry$country[which.min(lifeExp_bycountry$mean_lifeExp)]
+> >```
+> Another way to do this is to use the `dplyr` function `arrange()`, which has
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd]: I might add a bit on what `arrange()` does in this sentence.

Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd]: Also I might also might modify the challenge itself to clarify what is being asked for - is it the min/max individual value or the lowest/highest mean value that we'd like to find?

File: _episodes_rmd/13-dplyr.Rmd [hunk 2]
```diff
@@ -181,6 +181,17 @@ even better.
...
 > >    summarize(mean_lifeExp=mean(lifeExp))
+> >
+> >lifeExp_bycountry$country[which.max(lifeExp_bycountry$mean_lifeExp)]
+> >lifeExp_bycountry$country[which.min(lifeExp_bycountry$mean_lifeExp)]
+> >```
+> Another way to do this is to use the `dplyr` function `arrange()`, which has 
+> similar syntax to other functions from the `dplyr` package.
+> >```{r}
+> >lifeExp_bycountry %>%
+> >    arrange(mean_lifeExp)
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd]: Also could pipe into a `head(1)` call to each of these to get just the min and the max rows.

File: _episodes_rmd/13-dplyr.Rmd [hunk 3]
```diff
@@ -181,6 +181,17 @@ even better.
 > >lifeExp_bycountry <- gapminder %>%
 > >    group_by(country) %>%
 > >    summarize(mean_lifeExp=mean(lifeExp))
+> >
+> >lifeExp_bycountry$country[which.max(lifeExp_bycountry$mean_lifeExp)]
+> >lifeExp_bycountry$country[which.min(lifeExp_bycountry$mean_lifeExp)]
```
Comment (naupaka) [inline _episodes_rmd/13-dplyr.Rmd]: Might take these out since we're focusing on the dplyr approach in this episode.

**Comment by liz-is** [comment]
I've made the changes you suggested, and added @fmichonneau's suggested solution as well. 

Do you think it would be better to move the information about `arrange()` to the start of the challenge, or is it okay where it is?

**Comment by naupaka** [comment]
I think it looks good as is. Thanks for making the changes so quickly. Addresses #230

**Pull request merged**